doc/user/content/sql/alter-cluster.md

```diff
@@ -6,17 +6,56 @@ menu:
     parent: 'commands'
 ---
 
-`ALTER CLUSTER` changes the configuration of a cluster. To rename a
-cluster, use [`ALTER ... RENAME`](/sql/alter-rename/).
+`ALTER CLUSTER` changes the configuration of a cluster, which **might incur
+downtime**. To rename a cluster, use [`ALTER ... RENAME`](/sql/alter-rename/).
 
 ## Syntax
 
+#### Set new configuration value
+
 {{< diagram "alter-cluster-set.svg" >}}
 
+#### Reset configuration to original value
+
 {{< diagram "alter-cluster-reset.svg" >}}
 
+### Cluster options
+
 {{% cluster-options %}}
 
+### `with_options`
+
+{{< diagram "alter-cluster-set-with-options.svg" >}}
+
+Field                         | Value                 | Description
+------------------------------|-----------------------|-------------------------------------
+**WAIT UNTIL READY...TIMEOUT**                   | `duration`            | For a [graceful cluster resizing](#graceful-cluster-resizing), the maximum duration to wait for the new replicas to be ready.
+**WAIT UNTIL READY...ON TIMEOUT**                | [`COMMIT`,`ROLLBACK`] | For a [graceful cluster resizing](#graceful-cluster-resizing), the action to take on timeout. <br><ul><li>`COMMIT` cuts over to the new replica regardless of its hydration status, which may lead to downtime.</li><li>`ROLLBACK` removes the pending replica and returns a timeout error.</li></ul>Default: `COMMIT`.
+**WAIT FOR**                  | `duration`            | For a [graceful cluster resizing](#graceful-cluster-resizing), a fixed duration to wait for the new replicas to be ready. This option might lead to downtime, so we recommend using the `WAIT UNTIL READY` option instead.
```

Comment:
@jubrad, is there any scenario where we recommend using this option instead of `WAIT UNTIL READY`? This was missing from the option list, but I tried to de-emphasize it.

Reply:
I think the only time you would use this is if you wanted both sets of replicas up for the exact specified duration. This could be because you want do some sort of comparison between different sizes or something?

Reply:
Hum, I see. Might be too much work to explain if in the end we expect 99.9% of users to just hit a button in the Console to resize a cluster. I'd leave it as is, but deferring to @kay-kim.

Reply:
This works for me.